op-challenger/game/fault/preimages/large.go

```diff
@@ -34,7 +35,49 @@ func (p *LargePreimageUploader) UploadPreimage(ctx context.Context, parent uint6
 	// todo(proofs#467): split up the preimage into chunks and submit the preimages
 	//                   and state commitments to the preimage oracle contract using
 	//                   `PreimageOracle.addLeavesLPP` (`_finalize` = false).
+	uuid, err := p.newUUID()
+	if err != nil {
+		return fmt.Errorf("failed to generate UUID: %w", err)
+	}
```

Comment:
This makes sense for now, but lets make sure we don't depend on this being a random number - we'll almost certainly want to make it deterministic based on the content we're uploading so we can resume upgrades.  Actually working out how to do that can wait until https://github.com/ethereum-optimism/client-pod/issues/473 though.

Reply:
Makes sense! I'll add a `TODO` comment referring to that issue so it's clear in the upstreamed code.